lib/car.rb

```diff
@@ -0,0 +1,24 @@
+class Car
+  attr_reader :car, :make, :model, :monthly_payment,
+  :loan_length, :color
+  def initialize(make, model, loan_length)
+    @car = car
+    @make = "Ford"
+    @model = "Mustang"
+    @loan_length = 36
+    @monthly_payment = 1500
+    @color = nil
+  end
+
+  def total_cost
+    @monthly_payment * @loan_length
+  end
+
+  # def color
+  #   @car = nil
+  # end
+
+  def paint!(color)
+    @color = :blue
```

Comment:
You are assigning @color to `:blue` here instead of the `color` passed in via argument. Be careful - this may have passed a test you wrote, but if I sent it `:red` instead, it would always equal `:blue`!